bulldozer/config_fetcher.go

```diff
@@ -148,6 +148,20 @@ func (cf *ConfigFetcher) unmarshalConfig(bytes []byte) (*Config, error) {
 		return nil, errors.Wrapf(err, "failed to unmarshal configuration")
 	}
 
+	// Merge old signals configurations if they exist
+	if config.Merge.Blacklist != nil {
+		config.Merge.Denylist = *config.Merge.Blacklist
```

Comment:
I think this means that configuration setting both blacklist and denylist will use the values from the old fields. Is that the desired behavior or should the new fields win if both are set?

Reply:
You're right, I'll update this to only accept the legacy values when the new structs aren't present.